gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -117,21 +123,64 @@ def call_book(self, other_args):
         """Process book command"""
         binance_model.order_book(other_args, self.current_coin, self.current_currency)
 
-    def call_candle(self, other_args):
+    def call_candle(self, _):
         """Process candle command"""
-        binance_model.show_candles(other_args, self.current_coin, self.current_currency)
+        binance_model.show_candles(
+            self.current_df, self.current_coin, self.current_currency
+        )
 
     def call_balance(self, _):
         """Process balance command"""
         binance_model.balance(self.current_coin)
 
     def call_select(self, other_args):
         """Process select command"""
-        self.current_coin, self.current_currency = binance_model.select_binance_coin(
-            other_args
-        )
+        (
+            self.current_coin,
+            self.current_currency,
+            self.current_df,
+        ) = binance_model.select_binance_coin(other_args)
+        self.source = "BIN"
         print("")
 
+    # pylint: disable=inconsistent-return-statements
+    def call_ta(self, _):
+        """Process ta command"""
+        if not self.current_coin:
+            print("Please load a coin through either load or select", "\n")
+        if self.current_df.empty:
+            print("Price dataframe is empty")
+        else:
+            # Need to make the columns in df be compatible.  Also since there are no splits or dividends, there is no
+            # adj close.  To tell ta to use Close, we can just set the interval to anything but 1440.
```

Comment:
I don't understand this part.

Reply:
So the TA menu is meant to interface with the `load` command which uses the finance data frame.  If you look in terminal.py, there is this renaming of OHLCV columns to `1. open` etc.  I need to do that so that ta doesn't complain about `[5. adj close]`  not  being found